_includes/breadcrumbs.html

```diff
@@ -4,31 +4,39 @@
   {%- assign langSlug = site.baseurl | append: '/' | append: page.lang -%}
 {%- endif -%}
 
-
-<p class="content-center__home breadcrumbs">
-  
+<nav class="content-center__home breadcrumbs" aria-label="Je bent hier"> 
+  <ul> 
   {% for crumb in crumbs offset: 1 %}
   {{% assign start_crumb = '' %}}
     {% if forloop.first %}
-    <a class="breadcrumbs" href="{{langSlug}}">
-      {{ site.data.translations.layout-template-home[page.lang] }}
-    </a>
+    <li class="breadcrumbs-item"> 
+        <a class="breadcrumbs" href="{{langSlug}}">
+          {{ site.data.translations.layout-template-home[page.lang] }}
+        </a>
+        {% if forloop.last == false %}
+          <span class="icon icon-chevron-right"></span>
+        {% endif %}
+    </li> 
     {% elsif forloop.last == false %}
-      <a class="breadcrumbs" href="{{langSlug}}/faq">
-        {{ site.data.translations.faq[page.lang] | truncate: 40 }}
-      </a>
+      <li  class="breadcrumbs-item">
+        <a class="breadcrumbs" href="{{langSlug}}/faq">
+          {{ site.data.translations.faq[page.lang] | truncate: 40 }}
+        </a>
+        <span class="icon icon-chevron-right"></span>
+      </li>
     {% else %}
-      {% if page.contentLang == 'en' %}
-        <a class="breadcrumbs disabled" dir="ltr" lang="en">{{ include.title | truncate: 40 }}</a>
```

Comment:
I would suggest changing the `a` to `span` here. It is not really a link and a `span` reflects that better.
We could also add `aria-current=page` to indicate semantically that this is the current page in the breadcrumb.

Reply:
Thank you, I used your suggestion to improve this!